iowrappers/photos_client.go

```diff
@@ -41,7 +47,11 @@ type PhotoHttpClient struct {
 // CreatePhotoClient is a factory method for PhotoClient
 func CreatePhotoClient(apiKey string, baseURL string, enableMapPhotoClient bool) PhotoClient {
```

Comment:
Suggest return `(PhotoClient, error)`, otherwise the caller would not know if the client is valid.

Reply:
Sg, will do.

Reply:
Done, see [839254e]